### DISCUSSION TITLE
Can it support multiple languages?

### DISCUSSION THREAD

**Discussion by andy00614** [scalar/scalar#2458] [General]
I want to use scalar to generate API document in multiple languages. is it have a easy way to do it? Does the framework itself include this feature?

**Comment by hanspagel** [comment]
You mean a different language for descriptions and such? I don't know of a way to provide translations inside your OpenAPI document. 

We plan to support multiple specifications, though: #1851.

For now, the only way is to just have multiple API references I’d say.
